flake.nix

```
@@ -92,8 +92,8 @@
             prev.packages
             ++ (
               with common.pkgs;
-                [lld_13 lldb cargo-flamegraph rust-analyzer]
-                ++ (lib.optional (stdenv.isx86_64 && stdenv.isLinux) cargo-tarpaulin)
+                [lld_13 cargo-flamegraph rust-analyzer]
+                ++ (lib.optionals (stdenv.isx86_64 && stdenv.isLinux) [cargo-tarpaulin lldb])
```

Doh! I actually had the below change locally and forgot to make a PR:

```suggestion
                ++ (lib.optional (stdenv.isx86_64 && stdenv.isLinux) cargo-tarpaulin)
                ++ (lib.optional stdenv.isLinux lldb)
```

I _think_ lldb is only busted on darwin, so we don't need the x86_64 check: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/llvm/13/lldb/default.nix#L105